### PR TITLE
[Rails 6.1] Fix Deprecation Warning Typo

### PR DESCRIPTION
**Pull request by JuanVqz**
- Fix deprecation warning typo

### Motivation / Background

This Pull Request has been created because there is a typo in a deprecation warning message

### Detail

This Pull Request fixes a deprecation warning typo.

### Checklist

Before submitting the PR make sure the following are checked:

* [x] This Pull Request is related to one change. Changes that are unrelated should be opened in separate PRs.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [ ] Tests are added or updated if you fix a bug or add a feature.
* [ ] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.

### CODE REVIEW

**Comment by JuanVqz** [comment]
I haven't done this before, my apologies if this is not the way to proceed, if you point how should I proceed will be nice :)

File: actionpack/lib/action_dispatch/http/response.rb [hunk 1]
```diff
@@ -88,13 +88,13 @@ def each(&block)
 
     def self.return_only_media_type_on_content_type=(*)
       ActiveSupport::Deprecation.warn(
-        ".return_only_media_type_on_content_type= is dreprecated with no replacement and will be removed in 7.0."
+        ".return_only_media_type_on_content_type= is deprecated with no replacement and will be removed in 6.2."
```
Comment (guilleiguaran) [inline actionpack/lib/action_dispatch/http/response.rb:91]: The 7.0 -> 6.2 change is wrong, we don't have plans to release a 6.2 version.

Comment (JuanVqz) [reply actionpack/lib/action_dispatch/http/response.rb:91]: Thank you for letting me know, and updating it.

**Comment by JuanVqz** [comment]
Closed in favor of https://github.com/rails/rails/pull/47319

**Pull request closed without merging**